src/Factory.php

```diff
@@ -51,7 +50,9 @@ public function createPhiremockClient(Host $host, Port $port): Phiremock
             $this->createRemoteConnection(),
             $this->phiremockFactory->createV2UtilsFactory()->createExpectationToArrayConverter(),
             $this->phiremockFactory->createV2UtilsFactory()->createArrayToExpectationConverter(),
-            $this->phiremockFactory->createV2UtilsFactory()->createScenarioStateInfoToArrayConverter()
+            $this->phiremockFactory->createV2UtilsFactory()->createScenarioStateInfoToArrayConverter(),
+            $this->phiremockFactory->findRequestFactoryInterface(),
```

Comment:
These methods don't exist in phiremock-common. 
I see you created an extended version here, but since server and client use laminas the dependency for http-factory and discovery should probably be there.